unfurl/server.py

```diff
@@ -139,6 +142,78 @@ def export():
 
     return jsonify(json_summary)
 
+
+@app.route("/update_deployment", methods=["POST"])
```

Comment:
Let's add an optional parameter call "commit_msg" parameter that is used for the commit message instead of just "Update deployment"